extensions/google-drive/package.json

```diff
@@ -58,15 +60,20 @@
   ],
   "dependencies": {
     "@raycast/api": "^1.33.2",
+    "download": "^8.0.0",
     "fzf": "^0.5.1",
-    "sql.js": "^1.6.2"
+    "sql.js": "^1.6.2",
+    "tar": "^6.1.11"
   },
   "devDependencies": {
+    "@types/download": "^8.0.1",
     "@types/node": "~16.10.0",
     "@types/react": "^17.0.28",
     "@types/sql.js": "^1.4.3",
+    "@types/tar": "^6.1.1",
     "@typescript-eslint/eslint-plugin": "^5.0.0",
     "@typescript-eslint/parser": "^5.0.0",
+    "electron": "^18.2.0",
```

Comment:
The `download` package doesn't work otherwise.

Reply:
How big is the `electron` package? I imagine this is a rather large download to include in the extension.

Reply:
I don't like it either. I will actually kill the `download` package later.